module-system/sov-modules-core/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Can we also have `#![deny(missing_docs)]` here, so we don't have to do it again.

Reply:
Good suggestion; updated